src/averageAges.js

```diff
@@ -55,7 +64,16 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findParents = (onlyWithSon
```

Comment:
```suggestion
  const parents = (onlyWithSon
```
findParents sounds like function, but it's array